packages/artifact_proxy/lib/config.dart

```diff
@@ -3,10 +3,41 @@ import 'package:artifact_proxy/artifact_proxy.dart';
 /// The proxy configuration used by the server.
 const config = ProxyConfig(
   engineMappings: {
+    ...flutter_3_7_9,
     ...flutter_3_7_8,
   },
 );
 
+/// Flutter 3.7.9
+const flutter_3_7_9 = {
+  '2b9e2639de0a81e55b4640465facc448d5e9c269': EngineMapping(
```

Comment:
This revision is still 3.7.8, just with support for more platforms?

Reply:
I thought it was 3.7.9 based on the git log